crates/core/tedge/src/cli/mqtt/mod.rs

```diff
@@ -11,7 +11,7 @@ mod error;
 mod publish;
 mod subscribe;
 
-const MAX_PACKET_SIZE: usize = 10 * 1024 * 1024;
+const MAX_PACKET_SIZE: usize = 268435455; // 256 MB
```

Comment:
I would not change the default, now that the correct protection is implemented.

```suggestion
    /// Default: `1024 * 1024`.
```

Reply:
Resolved by f43495ebc367685bb9cfc8fafdc9d445a8eea263